src/main/java/com/kamesuta/bungeepteropower/PteroCommand.java

```diff
@@ -71,17 +76,40 @@ public void execute(CommandSender sender, String[] args) {
                         ? PterodactylAPI.PowerSignal.START
                         : PterodactylAPI.PowerSignal.STOP;
 
+                // Cancel existing stop task
+                if (signal == PterodactylAPI.PowerSignal.STOP)
+                    plugin.delay.cancelStop(serverName);
+
+                boolean autoJoin = subCommand.equals("start") && args.length >= 3 && args[2].equals("join") && sender instanceof ProxiedPlayer;

```

Comment:
It would be simpler and easier to use if the autoJoinTimeout was set to 1 or more in the config, and the autojoin function was always enabled.